packages/inter-protocol/test/auction/test-scheduler.js

```diff
@@ -654,3 +654,101 @@ test('duration = freq', async t => {
   };
   t.deepEqual(schedule.nextAuctionSchedule, secondSchedule);
 });
+
+test('change Schedule', async t => {
+  const { zoe } = await setupZCFTest();
+  const installations = await setUpInstallations(zoe);
+  /** @type {TimerService & { advanceTo: (when: Timestamp) => void; }} */
+  const timer = buildManualTimer();
+  const timerBrand = await timer.getTimerBrand();
+
+  const fakeAuctioneer = makeFakeAuctioneer();
+  const { fakeInvitationPayment } = await getInvitation(zoe, installations);
+  const publisherKit = makeGovernancePublisherFromFakes();
+
+  const { makeRecorderKit, storageNode } = prepareMockRecorderKitMakers();
+  const recorderKit = makeRecorderKit(storageNode);
+  let defaultParams = makeDefaultParams(fakeInvitationPayment, timerBrand);
+  // start hourly, request 6 steps down every 10 minutes, so duration would be
+  // 1 hour. Instead, cut the auction short.
+  defaultParams = {
+    ...defaultParams,
+    priceLockPeriod: 20n,
+    startFreq: 360n,
+    auctionStartDelay: 5n,
+    clockStep: 60n,
+    startingRate: 100n,
+    lowestRate: 40n,
+    discountStep: 10n,
+  };
+  const params = makeAuctioneerParams(defaultParams);
+  const params2 = {};
+  for (const key of Object.keys(params)) {
+    const { value } = params[key];
+    params2[key] = value;
+  }
```

Comment:
see comment below on the type. some feedback here that `params2` is a  poor name since it's transforming `params` in a specific way. I think we have a utility already for that transformation but I can't find it. It would be handy to have in the governance package as an exported tool. 

at the least,
```js
const paramValues = objectMap(makeAuctioneerParams(defaultParams), r => r.value);
```

Reply:
done